sources/models/src/modeled_types/mod.rs

```diff
@@ -145,6 +145,18 @@ pub mod error {
         #[snafu(display("Invalid ECS duration value '{}'", input))]
         InvalidECSDurationValue { input: String },
 
+        #[snafu(display(
+            "Invalid OCI Default value '{}': {}. Validity rule: {}",
+            default_value_name,
+            provided_value,
+            validity_rule,
+        ))]
+        InvalidOciDefaults {
+            default_value_name: String,
+            provided_value: String,
+            validity_rule: String,
+        },
+
```

Comment:
Maybe I'm missing something but I don't see this error ever being used.

Reply:
You are correct - removed